main_test.go

```diff
@@ -26,15 +26,18 @@ func TestRunsSuite(t *testing.T) {
 	//	dns.SetManifestPath("testdata/my-custom-solver"),
 	//	dns.SetBinariesPath("_test/kubebuilder/bin"),
 	//)
-
+	os.Setenv("TEST_ASSET_ETCD", "_test/kubebuilder/bin/etcd")
+	os.Setenv("TEST_ASSET_KUBE_APISERVER", "_test/kubebuilder/bin/kube-apiserver")
+	defer os.Unsetenv("TEST_ASSET_ETCD")
+	defer os.Unsetenv("TEST_ASSET_KUBE_APISERVER")
 	solver := example.New("59351")
 	fixture := dns.NewFixture(solver,
 		dns.SetResolvedZone("example.com."),
 		dns.SetManifestPath("testdata/my-custom-solver"),
-		dns.SetBinariesPath("_test/kubebuilder/bin"),
 		dns.SetDNSServer("127.0.0.1:59351"),
 		dns.SetUseAuthoritative(false),
 	)
 
-	fixture.RunConformance(t)
```

Comment:
Can we keep this calling the full RunConformance function too?

Reply:
the RunConformance calls setup and then RunBasic and RunExtended call setup again and we get an error that dns server already running

Reply:
@roytev indeed, this probably should be changed in cert-manager first?
I also wonder why these setup calls are deferred, @munnerz can you help?
https://github.com/cert-manager/cert-manager/blob/b1180c59ad588e73ac25b0d70a86661cf7c180e1/test/acme/dns/fixture.go#L109-L129

Reply:
@roytev I'll create a PR to fix that bug